packages/react-native/ReactCommon/react/renderer/attributedstring/conversions.h

```diff
@@ -243,6 +243,35 @@ inline void fromRawValue(
     return;
   }
 
+  if(value.hasType<int>()) {
+    auto numeric = (int)value;
+    if (numeric == 100) {
+      result = FontWeight::Weight100;
+    } else if (numeric == 200) {
+      result = FontWeight::Weight200;
+    } else if (numeric == 300) {
+      result = FontWeight::Weight300;
+    } else if (numeric == 400) {
+      result = FontWeight::Weight400;
+    } else if (numeric == 500) {
+      result = FontWeight::Weight500;
+    } else if (numeric == 600) {
+      result = FontWeight::Weight600;
+    } else if (numeric == 700) {
+      result = FontWeight::Weight700;
+    } else if (numeric == 800) {
+      result = FontWeight::Weight800;
+    } else if (numeric == 900) {
+      result = FontWeight::Weight900;
+    } else {
+      LOG(ERROR) << "Unsupported FontWeight value: " << numeric;
+      react_native_expect(false);
```

Comment:
This react_native_expect doesn't log anything useful. You can remove this one and the one on L239.